apps/site/assets/ts/schedule/__tests__/MapTest.tsx

```diff
@@ -95,7 +97,7 @@ describe("reducer", () => {
 
   it("adds vehicles", () => {
     const result = reducer(
-      { markers: data.markers, channel: "vehicle:1:1", shapeId: "1" },
+      { markers: data.markers, channel: "vehicle:1:1", shapeIds: ["1"] },
```

Comment:
This may be out of scope, but shapeIds in this PR (and section of the codebase) appear to be strings containing integers. Is there a reason they aren't just regular integers, or a specific Shape ID type?

Reply:
Generally they're not integers, one example (a particularly complicated one) of a non-integral ID is `933_0010-70105-70102-2`. You can see more examples by going to e.g. https://api-v3.mbta.com/shapes?filter[route]=Red.

A ShapeID type would be a good idea but I don't see it as in scope of this PR. Since you bring it up, though, it would be good for us to make a practice out of introducing intention-revealing types in new TS code.

Reply:
Ah yeah, ok. I missed that. Definitely out of scope.